pyodide_build/pywasmcross.py

```diff
@@ -316,10 +348,10 @@ def install_for_distribution(args):
     try:
         subprocess.check_call(commands)
     except Exception:
-        print(f'Warning: {" ".join(commands)} failed with distutils, possibly '
-              f'due to the use of distutils that does not support the '
-              f'--old-and-unmanageable argument. Re-trying the install '
-              f'without this argument.')
+        print(f'Warning: {" ".join(str(arg) for arg in commands)} failed '
```

Comment:
There was an issue here because `arg[0]` was a PosixPath and a str was expected.